gulpfile.js/tests/configGenerator.test.js

```diff
@@ -0,0 +1,123 @@
+var fs = require('fs')
+var test = require('tape')
+var configGenerator = require('./../util/backstop/configGenerator')
+var deepClone = require('./../util/deepClone')
+var testFixturesPath = './gulpfile.js/tests/fixtures/configGenerator/'
+var config = {
+  compLib: {
+    port: 7000,
+    host: 'http://example',
+    baseDir: testFixturesPath
+  },
+  vrt: {
+    backstopConfigTemplate: testFixturesPath + 'backstop.template.json',
+    backstopConfig: testFixturesPath + 'backstop.json'
+  }
+}
+
+var removeFiles = function (files) {
+  return new Promise(function (resolve) {
+    files.forEach(function (file) {
+      fs.unlink(file, resolve)
+    })
+  })
+}
+
+var getBackstopConfigData = function () {
+  return new Promise(function (resolve, reject) {
+    fs.readFile(config.vrt.backstopConfig, function (err, data) {
```

Comment:
So `fixtures/configGenerator/backstop.json` is generated by the tests (and torn down) so I wanted to be able to error check this as it potentially may not be there. I could add it as a `require` but I would have to wrap that in a `try catch` incase its not there. I'm also not to sure I like a require in a function, rather than at the top of the page.